string name
geometry_msgs/PoseStamped pose
sensor_msgs/Image rgb_frame
sensor_msgs/Image depth_frame
string model_path
float32 scale
